dvc.yaml

```
@@ -0,0 +1,18 @@
+stages:
+  create_model:
+    cmd: python src/create_model.py
```

```suggestion
    cmd: pipenv run create
```
And then we need `create = "python -m src.create_model"` in the `[scripts]` sections in the Pipfile